components/HomepageSection/Activity.jsx

```diff
@@ -60,7 +60,7 @@ export const Activity = () => {
             {metrics?.a2aTransactions ? (
               <ExternalLink
                 className="text-2xl font-bold text-purple-600"
-                href="https://flipsidecrypto.xyz/flipsideteam/q/PVARr5q0B7HD/mech-requests/visualizations/v2/34079b3c-8115-428b-9902-5a0afca3149e"
+                href="https://dune.com/queries/5204254"
```

Comment:
could be this maybe instead of the table? https://dune.com/queries/5204254/8561534